public/static/locales/en/manageProjects.json

```diff
@@ -91,5 +91,14 @@
   "backToSpending": "Back to project spending",
   "pendingReview": "Pending Review",
   "submitForReview": "Submit for Review",
-  "addProject": "Add Project"
+  "addProject": "Add Project",
+  "abandonment": "Abandonment",
+  "abandonmentInfo": "Abandonment Information",
+  "survivalRate": "Survival Rate",
+  "survivalRateInfo": "Survival Rate Information",
```

Comment:
This is probably wrong at should be replace with this like already live at https://beta.plant-for-the-planet.org/trees-for-peace-an-ayudh-europe-campaign
<img width="182" alt="Bildschirmfoto 2020-11-26 um 10 18 21" src="https://user-images.githubusercontent.com/1532418/100331880-45433680-3003-11eb-95dd-2803bdb90ec5.png">